app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class CommentsController < ApplicationController
+  def index
+    @comments = @post.comments.order(created_at: :desc)
```

Comment:
- [ ] Going to `http://localhost:3000/posts/1/comments` gives `undefined method comments for nil class`. That's because the `@post` at this line is nill we need to get it first from the params.